src/controllers/userController.js

```diff
@@ -0,0 +1,35 @@
+import { Subscribes } from "../database";
+
+class userController {
+
+    static async subscribe(req,res){
+        const {
+            username,
+            email
+        }= req.body;
+        try {
+          const users = await Subscribes.where('email','==',email).get();
+          if(!users.empty){
+              return res.status(201).json({
+                status: 201,
+                message: "you are already subscribed",
+              });
+          }
+
+          await Subscribes.doc().set({username,email}).then(()=>{
+                res.status(200).json({
+                  message: "Subscrition went well",
```

Comment:
I wish that the user should receive an email notification for his/her successful subscription.